parsons/hustle/hustle.py

```diff
@@ -113,6 +115,103 @@ def _error_check(self, r, raise_on_error):
             logger.info(r.json())
             return None
 
+    def get_agents(self, group_id):
+        """
+        Get a list of agents.
+
+        `Args:`
+            group_id: str
+                The group id.
+
+        `Returns:`
+            Parsons Table
+                See :ref:`parsons-table` for output options.
+        """
+
+        tbl = Table(self._request(f'groups/{group_id}/agents'))
+        logger.info(f'Got {tbl.num_rows} agents from {group_id} group.')
+        return tbl
+
+    def get_agent(self, agent_id):
+        """
+        Get a single agent.
+
+        `Args:`
+            agent_id: str
+                The agent id.
+        `Returns:`
+            dict
+        """
+
+        r = self._request(f'agents/{agent_id}')
+        logger.info(f'Got {agent_id} agent.')
+        return r
+
+    def create_agent(self, group_id, name, full_name, phone_number, send_invite=False, email=None):
+        """
+        Create an agent.
+
+        `Args:`
+            group_id: str
+                The group id to assign the agent.
+            name: str
+                The name of the agent.
+            full_name: str
+                The full name of the agent.
+            phone_number: str
+                The valid phone number of the agent.
+            send_invite: boolean
+                Send an invitation to the agent.
+            email:
+                The email address of the agent.
+        `Returns:`
+            dict
+        """
+
+        agent = {'name': name,
+                 'fullName': email,
+                 'phoneNumber': phone_number,
+                 'sendInvite': send_invite,
+                 'email': email}
+
+        # Remove empty args in dictionary
+        agent = json_format.remove_empty_keys(agent)
+
+        logger.info(f'Generating {full_name} agent.')
+        return self._request(f'groups/{group_id}/agents', req_type="POST", payload=agent)
+
+    def update_agent(self, agent_id, name=None, full_name=None, send_invite=False):
+        """
+        Update an agent.
+
+        `Args:`
+            agent_id: str
+                The agent id.
+            name: str
+                The name of the agent.
+            full_name: str
+                The full name of the agent.
+            phone_number: str
```

Comment:
Looks like a copy-paste error.. this docstring describes Args for `phone_number` and `email`, but they are not valid function arguments.